ci-operator/step-registry/ovn/sdn-migration/feature-migration/config-egress-firewall/ovn-sdn-migration-feature-migration-config-egress-firewall-commands.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+set -x
+set -o errexit
+set -o nounset
+set -o pipefail
+
+oc create ns test-migration||true
```

Comment:
Why is the `||true` necessary? Why do we expect the possibility that `oc create ns test-migration` may fail?